lib/linter-integration.js

```diff
@@ -0,0 +1,31 @@
+'use babel';
+
+export default class LinterIntegration {
+  constructor(linterInstance) {
+    this.linter = linterInstance;
+    linterInstance.addLinter(this.provider);
+  }
+
+  get provider() {
+    return {
+      grammarScopes: [],
```

Comment:
Name gets a little tricky for Build. I'll figure it out, eventually. I want to have the name be dynamically set for the actual build provider but that seems to be a little tricky. But I wasn't worried about it for this first basic implementation because the `name` field isn't required yet.